web/pdf_cursor_tools.js

```diff
@@ -0,0 +1,182 @@
+/* Copyright 2016 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+(function (root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define('pdfjs-web/pdf_cursor_tools', ['exports', 'pdfjs-web/grab_to_pan',
+      'pdfjs-web/preferences'], factory);
+  } else if (typeof exports !== 'undefined') {
+    factory(exports, require('./grab_to_pan.js'), require('./preferences.js'));
+  } else {
+    factory((root.pdfjsWebPdfCursorTools = {}), root.pdfjsWebGrabToPan,
+      root.pdfjsWebPreferences);
+  }
+}(this, function (exports, grabToPan, preferences) {
+
+var GrabToPan = grabToPan.GrabToPan;
+var Preferences = preferences.Preferences;
+
+var CursorTool = {
+  SELECT: 0, // The default value.
+  HAND: 1,
+  ZOOM: 2,
```

Comment:
I'd actually like to keep it, to avoid having to touch this code if/once a zoom tool is added.